annotations.py

```diff
@@ -158,6 +162,7 @@ def to_HTML(self, header=''):
 #         self._log_location("comments_body='%s'" % comments_body)
 
         if self.annotations:
+            html_color_pattern = re.compile("^#(?:[0-9a-fA-F]{3,4}){1,2}$")
```

Comment:
Please remove code (across this PR), that is now extracted into separate PRs like #24.

Reply:
This will land last and I'll rebase, I don't need to remove it here. I'd wait to review this PR until those others merge.

Reply:
OK.